src/data_inclusion/scripts/tasks/sources/mes_aides.py

```diff
@@ -0,0 +1,119 @@
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytz
+from dateutil.parser import parse as dateutil_parse
+from pyairtable import Table
+
+from data_inclusion.schema import models
+
+MES_AIDES_SOURCE_STR = "mes-aides"
+
+
+def extract_data(api_key: str, base_id: str, table_name: str) -> Path:
```

Comment:
```suggestion
def extract_data(src: str, token: str, **kwargs) -> Path:
```

en fait il faut respecter la même interface que dans les autres `extract_data`

où `src` est l'url vers la source et peut provenir par exemple de la cli `data_inclusion preprocess <SRC_TYPE> <SRC_URL> --token <TOKEN>` (cf `data_inclusion/scripts/entrypoints/cli.py:33`).
idem pour `token`

Reply:
tu peux extraire `base_id` et `table_id` depuis l'url de la table sur airtable : cf https://support.airtable.com/docs/understanding-airtable-ids